fabric-input-events-v0/src/main/java/net/fabricmc/fabric/mixin/event/input/client/InputUtilTypeMixin.java

```diff
@@ -0,0 +1,14 @@
+package net.fabricmc.fabric.mixin.event.input.client;
+
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.gen.Accessor;
+
+import it.unimi.dsi.fastutil.ints.Int2ObjectMap;
+import net.minecraft.client.util.InputUtil;
+import net.minecraft.client.util.InputUtil.Key;
+
+@Mixin(InputUtil.Type.class)
+public interface InputUtilTypeMixin {
+	@Accessor("map")
+	public Int2ObjectMap<Key> fabric_getMap();
```

Comment:
```
	public Int2ObjectMap<Key> fabric_getMap();
```

No need for `fabric_` prefix when dealing with accessor mixins. Only case you should use it really is it there is a conflict and such conflict

Reply:
> Only case you should use it really is it there is a conflict and such conflict

Doesn't every mixin expose inject their symbols into the target class? I'm don't think it's needed here, but what about the `@ModifyVariable`s?